core/lib/presentation/utils/html_transformer/html_utils.dart

```diff
@@ -97,6 +97,31 @@ class HtmlUtils {
     </script>
   ''';
 
+  static const lineHeight100Percent = (
+    script: '''
+      document.querySelectorAll("*")
+        .forEach((element) => {
+          if (element.style.lineHeight !== "normal")
+            element.style.lineHeight = "100%";
```

Comment:
IMO, we will be affected when some tags have a specific `lineHeight` set (like set via rich text button).
### Propose

Way 1: Keep the content of the text intact when `copy & paste` or `drag & drop` into the editor.
Way 2: Only change `lineHeight` for content to be `dragged` or `copied` in the editor

Reply:
Will implement option 2

Reply:
I can't seem to find a way to get the `dragged` or `copied` content. Suggesting to keep it this way & fix bug later if it ever occurs